Kickstarter-iOS/Features/ChangeEmail/Controller/ChangeEmailView.swift

```diff
@@ -0,0 +1,195 @@
+import Library
+import SwiftUI
+
+enum FocusField {
+  case newEmail
+  case currentPassword
+}
+
+@available(iOS 15.0, *)
+struct ChangeEmailView: View {
+  @State var emailText: String
+  @State private var newEmailText = ""
+  @State private var passwordText = ""
+  @SwiftUI.Environment(\.defaultMinListRowHeight) var minListRow
+  @FocusState private var focusField: FocusField?
```

Comment:
If this property wrapper is the only iOS 15 specific feature that we're using here (it might not be), maybe we omit this in favor of getting this view out to everyone rather than just those on that version?

We could setup @Published property wrappers in the VM to handle focus states in a similar way to handle iOS 14 members. Its not complicated but would require some extra boilerplate code. Probably via a `UIViewRepresentable` in order to access the `didBecomeFirstResponder` type properties

Reply:
will address in this [PR](https://github.com/kickstarter/ios-oss/pull/1803)